### PR TITLE
Add German major (GER)

### PR DESCRIPTION
**Pull request by bcatoto**

### CODE REVIEW

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+                    "GER 2*"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Politics",
+              "max_counted": 1,
+              "min_needed": 4,
+              "explanation": "<i>Editor's Note: Most of these courses fall under this requirement but as TigerPath does not verify this.</i>\n\nFour courses will be taken in the Department of Politics (usually no more than two at the 200 level), on topics pertaining to German/European politics or political theory.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]: Is it true that _most_ of the courses satisfy the requirement, or is it only _some_? 
Also there is an extra "as" in the editor's note.

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+                    "GER 2*"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Cognates",
+              "max_counted": 1,
+              "min_needed": 3,
+              "explanation": "<i>Editor's Note: Most of these courses fall under this requirement but as TigerPath does not verify this.</i>\n\nThree upper-level courses in a second literature.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]: Same here (see my comment on line 358)

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+          "min_needed": "ALL",
+          "explanation": "Concentrates on the history and structure of the German language. Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics, and two seminars related to topics in Germanic linguistics: GER 505 (History of the German Language) or GER 508, and GER 506 (Second Language Acquisition and Pedagogy) or GER 316. In addition, such students will take at least three other courses in German literature and culture (usually no more than two at the 200 level), as well as two cognate courses.",
+          "req_list": [
+            {
+              "name": "Linguistics",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics.",
+              "course_list": [
+                "LIN 213",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]: This seems like a typo on the department's end, right?
"Introduction to Language and Linguistics" is actually LIN 201, while LIN 213 is "Languages and Culture in South Asia" which seems quite irrelevant.
```suggestion
                "LIN 201",
```

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+                  "explanation": null,
+                  "course_list": [
+                    "GER 506",
+                    "GER 316"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "German",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]:
```suggestion
              "name": "German literature and culture",
```
https://github.com/TigerPathApp/tigerpath/issues/189#issuecomment-654579852

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 5]
```diff
@@ -0,0 +1,403 @@
...
+                    "GER 2*"
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "Cognates",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Two cognate courses.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]: They are more explicit in the [handbook](https://german.princeton.edu/sites/default/files/2019-11/2019-2020%20current%20Undergraduate%20Handbook%20Nov.pdf).
```suggestion
              "explanation": "Two cognate courses pertaining to linguistics.",
```

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 6]
```diff
@@ -0,0 +1,403 @@
+{
+  "type": "Major",
+  "name": "German",
+  "code": "GER",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-german",
+    "https://german.princeton.edu/programs/undergraduate/about-major"
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]: Add the handbook as well
```suggestion
    "https://german.princeton.edu/programs/undergraduate/about-major",
    "https://german.princeton.edu/sites/default/files/2019-11/2019-2020%20current%20Undergraduate%20Handbook%20Nov.pdf"
```

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 7]
```diff
@@ -0,0 +1,403 @@
...
+          "name": "German Culture and Politics",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Students may combine a concentration in German literature and culture with a concentration in German/European politics and political theory. Four courses will be taken in the German Department (usually no more than two at the 200 level), on topics pertaining to German literature and culture, and four courses will be taken in the Department of Politics (usually no more than two at the 200 level), on topics pertaining to German/European politics or political theory. The senior thesis may focus on any political topic with a substantive German-related component. Upon graduation, a letter will be issued by the department certifying the completion of a program in German cultural studies with a concentration in politics.",
+          "req_list": [
+            {
+              "name": "German",
+              "max_counted": 1,
+              "min_needed": 4,
+              "explanation": "Four courses will be taken in the German Department (usually no more than two at the 200 level), on topics pertaining to German literature and culture.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]: The [handbook](https://german.princeton.edu/sites/default/files/2019-11/2019-2020%20current%20Undergraduate%20Handbook%20Nov.pdf) provides some specific course recommendations, so why not provide the info. 
In fact we might consider switching to instead having a course list with these specific courses, but I guess it's also fine as is.
```suggestion
              "explanation": "Four courses will be taken in the German Department (usually no more than two at the 200 level), on topics pertaining to German literature and culture. Recommended departmental courses are GER 207, 208, 211, 306, 307, 309, and 324.",
```

File: tigerpath/majors_and_certificates/majors/GER_2018.json (hunk 1 above)
Comment (bnehoran) [reply tigerpath/majors_and_certificates/majors/GER_2018.json]: Same as above (see comment on line 330), the [handbook](https://german.princeton.edu/sites/default/files/2019-11/2019-2020%20current%20Undergraduate%20Handbook%20Nov.pdf) provides some specific course recommendations.

Comment (bnehoran) [reply tigerpath/majors_and_certificates/majors/GER_2018.json]:
```suggestion
              "explanation": "<i>Editor's Note: Not all of these courses fall under this requirement and TigerPath does not verify this.</i>\n\nFour courses will be taken in the Department of Politics (usually no more than two at the 200 level), on topics pertaining to German/European politics or political theory. Recommended cognates in politics include POL 210, 230, 240, 301, 303, 313, 316, 349, 385, and 388.",
```

File: tigerpath/majors_and_certificates/majors/GER_2018.json (hunk 2 above)
Comment (bnehoran) [reply tigerpath/majors_and_certificates/majors/GER_2018.json]:
```suggestion
              "explanation": "<i>Editor's Note: Not all of these courses fall under this requirement and TigerPath does not verify this.</i>\n\nThree upper-level courses in a second literature.",
```

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 8]
```diff
@@ -0,0 +1,403 @@
...
+                "MUS *",
+                "PHI *",
+                "ECS *",
+                "VIS *"
+              ]
+            }
+          ]
+        },
+        {
+          "name": "German Linguistics",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]: This seems to be the official name according to the department.
```suggestion
          "name": "Germanic Linguistics",
```

File: tigerpath/majors_and_certificates/majors/GER_2018.json (hunk 3 above)
Comment (bnehoran) [reply tigerpath/majors_and_certificates/majors/GER_2018.json]: Oh, it seems the [handbook](https://german.princeton.edu/sites/default/files/2019-11/2019-2020%20current%20Undergraduate%20Handbook%20Nov.pdf) got it right and says LIN 201, so this is almost definitely a typo.

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 9]
```diff
@@ -0,0 +1,403 @@
...
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Concentrates on the history and structure of the German language. Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics, and two seminars related to topics in Germanic linguistics: GER 505 (History of the German Language) or GER 508, and GER 506 (Second Language Acquisition and Pedagogy) or GER 316. In addition, such students will take at least three other courses in German literature and culture (usually no more than two at the 200 level), as well as two cognate courses.",
+          "req_list": [
+            {
+              "name": "Linguistics",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]: See my comment on line 194
```suggestion
              "explanation": "Majors who select this track are required to take Linguistics 201 (Introduction to Language and Linguistics) or a comparable course in linguistics.",
```

File: tigerpath/majors_and_certificates/majors/GER_2018.json [hunk 10]
```diff
@@ -0,0 +1,403 @@
...
+                "VIS *"
+              ]
+            }
+          ]
+        },
+        {
+          "name": "German Linguistics",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Concentrates on the history and structure of the German language. Majors who select this track are required to take Linguistics 213 (Introduction to Language and Linguistics) or a comparable course in linguistics, and two seminars related to topics in Germanic linguistics: GER 505 (History of the German Language) or GER 508, and GER 506 (Second Language Acquisition and Pedagogy) or GER 316. In addition, such students will take at least three other courses in German literature and culture (usually no more than two at the 200 level), as well as two cognate courses.",
```
Comment (bnehoran) [inline tigerpath/majors_and_certificates/majors/GER_2018.json]: See my comment on line 194
```suggestion
          "explanation": "Concentrates on the history and structure of the German language. Majors who select this track are required to take Linguistics 201 (Introduction to Language and Linguistics) or a comparable course in linguistics, and two seminars related to topics in Germanic linguistics: GER 505 (History of the German Language) or GER 508, and GER 506 (Second Language Acquisition and Pedagogy) or GER 316. In addition, such students will take at least three other courses in German literature and culture (usually no more than two at the 200 level), as well as two cognate courses.",
```

**Review by bnehoran — CHANGES_REQUESTED** [review]

**Review by bnehoran — DISMISSED** [review]
Looks great!

**Comment by bcatoto** [comment]
I made a small edit to Germanic Linguistics to fix something.

**Review by bnehoran — APPROVED** [review]
